babel.config.js

```diff
@@ -11,7 +12,7 @@ module.exports = function getBabelConfig(api) {
       '@babel/preset-env',
       {
         bugfixes: true,
-        browserslistEnv: process.env.BABEL_ENV || process.env.NODE_ENV,
+        browserslistEnv: process.env.NODE_ENV,
```

Comment:
Can this be `api.env()`? Two different flags (NODE_ENV and BABEL_ENV) made this file hard to reason about in core.

Reply:
Sure thing!